UBC-Meetable/code
# Automatically sign in after sign up

## Objective
This change allows users to be automatically signed in when they first sign up for a new account. Specifically, after they confirm their email by entering the code, they will be taken directly to the sign-up screen i.e. `UniScreen`.

- Passes `password` prop to `ConfirmEmailScreen` which is required for `Auth.signIn`.
- Set current `user` by calling `setUser` declared in `UserContext.tsx`.

Implements/Fixes [Make Signup Immediately log users in](https://app.asana.com/0/1200965514083070/1200966043588720/f)

## Testing instructions
The behavior can be tested manually by signing up with a valid email address. Make sure that you finish the signing-up process to ensure you can see the list of courses and user profile information on the main app.

## Discussion point
The `login` function is a duplicate of the same function declared in `LoginFormScreen` which should be addressed when the refactoring is to be done.

## Checklist
